src/components/compoundComponents/Tabs/TabsExample.tsx

```diff
@@ -0,0 +1,18 @@
+import Tabs from './Tabs';
+
+function TabsExample() {
+  return (
+    <Tabs>
+      <div className="flex">
+        <Tabs.Tab index={0}>Tab 1</Tabs.Tab>
```

Comment:
should this contain more than just tab 1, tabpanel 1 and so on?

Reply:
Don't get it what exactly you mean?

Reply:
just better examples than " tab 1" :D

Reply:
I'm open for proposals :-)

Reply:
hmm something like in our stories? https://marigold-latest.vercel.app/?path=/story/components-tabs--basic